src/palette.js

```diff
@@ -39,6 +39,7 @@ export const BUTTON_BG = 'linear-gradient(to bottom, #6f8aee, #4862e1)';
 export const BUTTON_BORDER = '1px solid #5a79e6';
 export const BUTTON_TXT_COLOR = '#597ce4';
 export const GRADIENT_BG = 'linear-gradient(to bottom, #6f8aee, #4862e1)';
+export const SHADOW_COLOR = '#eaeaea';
```

Comment:
How about creating a new color palette, just like `RED` is e.g. `GRAY`?

Reply:
I'm working on refactor of this consts on other branch